clients/ui/frontend/src/__tests__/cypress/cypress.config.ts

```diff
@@ -0,0 +1,122 @@
+import path from 'path';
+import fs from 'fs';
+import { defineConfig } from 'cypress';
+import coverage from '@cypress/code-coverage/task';
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore no types available
+import cypressHighResolution from 'cypress-high-resolution';
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore no types available
+import { beforeRunHook, afterRunHook } from 'cypress-mochawesome-reporter/lib';
+import { mergeFiles } from 'junit-report-merger';
+import { env, BASE_URL } from '~/src/__tests__/cypress/cypress/utils/testConfig';
+
+
+const resultsDir = `${env.CY_RESULTS_DIR || 'results'}/${env.CY_MOCK ? 'mocked' : 'e2e'}`;
```

Comment:
We need to be sure we carry over som of these env variables to either a `.env` file or a package.json command, where `CY_RESULTS_DIR` comes from? if we get the default, it might be great to document it somewhere.